ws/Cargo.toml

```diff
@@ -12,4 +12,4 @@ documentation = "https://paritytech.github.io/jsonrpc/json_ws_server/index.html"
 log = "0.3"
 jsonrpc-core = { version = "7.0", path = "../core" }
 jsonrpc-server-utils = { version = "7.0", path = "../server-utils" }
-ws = { git = "https://github.com/tomusdrw/ws-rs" }
+ws = { git = "https://github.com/faern/ws-rs", branch = "arbitrary-handshake-responses" }
```

Comment:
We still use a lock file in Parity, also this change would need to be ported to `parity-1.*` branch first. I think the worst that could happen is that the builds would be broken - but that's true regarding any dependency referenced from git.

Anyway the changes are merged to `tomusdrw/ws-rs`, @faern would you mind updating the PR to reference back this repo?